addons/sys_list/stringtable.xml

```diff
@@ -41,5 +41,26 @@
             <German>Zeigt Benachrichtigungen für Funk und Funkgeräte.</German>
             <Japanese>無線機と送信に関する通知を表示します。</Japanese>
         </Key>
+        <Key ID="STR_ACRE_sys_list_hintBgColor_displayName">
+            <English>Hint Background Color</English>
```

Comment:
I would rather use "Notification" instead of "Hint" (but keep string name the same as function also uses "hint").

Reply:
I agree, was just using the same name.